pandaserver/dataservice/adder_gen.py

```diff
@@ -19,219 +20,238 @@
 from pandaserver.dataservice import closer
 from pandaserver.taskbuffer import EventServiceUtils, JobUtils, retryModule
 
-_logger = PandaLogger().getLogger("Adder")
+_logger = PandaLogger().getLogger("adder")
```

Comment:
Remember the comment about log name changes. This one will break SLS and the filebeat configuration